prem_utils/connectors/prem.py

```diff
@@ -167,27 +166,5 @@ def chat_completion(
                     temperature=temperature,
                     top_p=top_p,
                 )
-        else:
-            if model == "mamba":
-                _base_url = "https://mambaphi.compute.premai.io/v1/chat/completions"
-                data = {
-                    "model": model,
-                    "messages": messages,
-                    "max_length": max_tokens,
-                    "temperature": temperature,
-                    "top_p": top_p,
-                }
-            else:
-                _base_url = f"https://{model}.compute.premai.io/mii/default"
-                data = {
-                    "prompts": [message["content"] for message in messages],
-                    "max_length": max_tokens,
-                    "temperature": temperature,
-                    "top_p": top_p,
-                }
-            _headers = {"Content-Type": "application/json", "Authorization": f"Bearer {self._api_key}"}
-            try:
-                response = requests.post(_base_url, json=data, headers=_headers)
-            except self._prem_errors as error:
-                raise error
-            return response.text
+        except self._prem_errors as error:
```

Comment:
On the Modal side, we didn't put any error handling. So this can be replaced with a generic Exception for now, but at one point we need to create a mapping of status codes for Modal too.

Reply:
ohh okay then I can remove this and just put `except Exception as error`

Reply:
Done, I removed the custom errors and replaced with generic `Exception`